pages/admin/scan/index.tsx

```diff
@@ -24,9 +24,20 @@ export default function Admin() {
     setCurrentScan(data);
   };
 
-  const handleScan = async (data, video: HTMLVideoElement, setVideoReady, setPaused, tick) => {
+  const handleScan = async (
+    data: string,
+    video: HTMLVideoElement,
+    setVideoReady,
+    setPaused,
+    tick,
+  ) => {
+    if (!data.startsWith('hack:')) {
+      setScanData(data);
+      setSuccess('Invalid hacker tag format...');
+      return;
+    }
     const query = new URL(`http://localhost:3000/api/scan`);
-    query.searchParams.append('id', user.id);
+    query.searchParams.append('id', data.replaceAll('hack:', ''));
```

Comment:
Not sure if appending "hack:" to any qr code is the most secure way of going about this

Reply:
Well, it's the only fix I could do with the 5 minute break I had lol. If you have any ideas please let me know, but at least this way we can differentiate between data.